lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -450,6 +450,18 @@ OpFoldResult AtenNeBoolOp::fold(ArrayRef<Attribute> operands) {
   return IntegerAttr::get(IntegerType::get(getContext(), 1), a != b);
 }
 
+//===----------------------------------------------------------------------===//
+// AtenSqueezeOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenSqueezeOp::fold(ArrayRef<Attribute> operands) {
```

Comment:
we also need a .mlir test for this.

Reply:
.mlir test case has been added for the folding case. Thanks :)